### ISSUE TITLE
Getting "KeyError: 'name'" error when trying to get group members.

### ISSUE THREAD

**Issue by ilgiznurgaliev** [pycontribs/jira#977]
Hi.

I'm trying to get all members of the `administrators` group using simple code below:

```
from jira import JIRA

USERNAME = '<username>'
API_TOKEN = '<token>'
SERVER = '<server>'
PROJECT = '<project_key>'

OPTIONS = {"server": SERVER}
jira = JIRA(options=OPTIONS, basic_auth=(USERNAME, API_TOKEN))
group = jira.group_members('administrators')
print(group)
```

But getting the following error:
```
Traceback (most recent call last):
  File "jira-audit.py", line 33, in <module>
    group = jira.group_members('administrators')
  File "C:\Users\username\AppData\Local\Programs\Python\Python37\lib\site-packages\jira\client.py", line 1011, in group_members
    result[user['key']] = {'name': user['name'],
KeyError: 'name'
```
Tried this API call directly in Chrome:
`https://<subdomain>.atlassian.net/rest/api/2/group/member?groupname=administrators`
and as I can see there is no name field at all:
```
{
  self: "https://<subdomain>.atlassian.net/rest/api/2/group/member? 
  includeInactiveUsers=false&maxResults=50&groupname=administrators&startAt=0",
  maxResults: 50,
  startAt: 0,
  total: 1,
  isLast: true,
  values: [
    {
      self: "...",
      accountId: "...",
      emailAddress: "...",
      avatarUrls: {...},
      displayName: "...",
      active: true,
      timeZone: "Europe/London",
      accountType: "atlassian"
    }
  ]
}
```

**Comment by andrewtmendoza** [comment]
I have the same issue

**Comment by LeoIannacone** [comment]
This fixes the issue:

jira/client.py:1010
```
        for user in r['users']['items']:
            result[user['accountId']] = {'name': user.get('name'),
                                   'fullname': user['displayName'],
                                   'email': user.get('emailAddress', 'hidden'),
                                   'active': user['active'],
                                   'accountId': user['accountId']}
```

**Comment by manc-lx** [comment]
@ilgiznurgaliev this has been fixed by #1079.

**Comment by adehad** [comment]
It has indeed, thanks again @manc-lx
